Libraries/Components/ScrollResponder.js

```diff
@@ -112,6 +114,27 @@ type State = {
 };
 type Event = Object;
 
+if (Platform.OS === 'android') {
+  var AndroidTextInput = requireNativeComponent('AndroidTextInput', null);
```

Comment:
I didn't realize native components can have many versions.

I don't have a strong opinion about this—just saying you can use `._currentElement.type` and it seemed safer to me than comparing strings. You could put a static flag on it, e.g. `_currentElement.type.isReactNativeInternalTextComponent` or something like this if you're worried about multiple definitions.